Content/Library/README.md

```diff
@@ -17,41 +17,85 @@ Stable | Prerelease
 
-### Watch Tests
 
-The `WatchTests` target will use [dotnet-watch](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/dotnet-watch.md) to watch for changes in your lib or tests and re-run your tests on all `TargetFrameworks`
+---
+
+### Building
+
 
 ```sh
-./build.sh WatchTests
+> build.cmd <optional buildtarget> // on windows
+$ ./build.sh  <optional buildtarget>// on unix
 ```
 
+The bin of your library should look similar to:
+
+```
+$ tree src/MyCoolNewLib/bin/
+src/MyCoolNewLib/bin/
+└── Debug
+    ├── net461
+    │   ├── FSharp.Core.dll
+    │   ├── MyCoolNewLib.dll
+    │   ├── MyCoolNewLib.pdb
+    │   ├── MyCoolNewLib.xml
+    └── netstandard2.1
+        ├── MyCoolNewLib.deps.json
+        ├── MyCoolNewLib.dll
+        ├── MyCoolNewLib.pdb
+        └── MyCoolNewLib.xml
+
+```
+
+---
+
+### Build Targets
+
+- `Clean` - Cleans up artifact and temp directories.
+- `DotnetRestore` - Runs [dotnet restore](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore?tabs=netcore2x) on the [solution file](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019).
+- [`DotnetBuild`](#Building) - Runs [dotnet build](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x) on the [solution file](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019).
+- `DotnetTest` - Runs [dotnet test](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?tabs=netcore21) on the [solution file](https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019).
+- `GenerateCoverageReport` - Code coverage is run during `DotnetTest` and this generates a report via [ReportGenerator](https://github.com/danielpalme/ReportGenerator)
+- `WatchTests` - Runs [dotnet watch](https://docs.microsoft.com/en-us/aspnet/core/tutorials/dotnet-watch?view=aspnetcore-3.0) with the test projects. Useful for rapid feedback loops.
+- `GenerateAssemblyInfo` - Generates [AssemblyInfo](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.applicationservices.assemblyinfo?view=netframework-4.8) for libraries 
+- `DotnetPack` - Runs [dotnet pack](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack). This include running [SourceLink](https://github.com/dotnet/sourcelink)
+- `SourcelinkTest` - Runs a Sourcelink Test tool to verify Sourcelinks were properly generated
+- `PublishToNuget` - Publishes the nuget via [paket push](https://fsprojects.github.io/Paket/paket-push.html)
+- `GitRelease` - Creates a commit message with the [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) and a git tag via the version in the `Release Notes`
+- `GitHubRelease` - Publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) with the Release Notes and any nuget packages
+- `FormatCode` - Runs [Fantomas](https://github.com/fsprojects/fantomas) on the sln
```

Comment:
Maybe add a brief description of Fantomas? Like `Runs Fantomas, a code formatter, on the solution`, so they don't need to click through the link to understand what this is from a high level